src/containers/Home/HomeHero/hello-world.ts

```diff
@@ -5,6 +5,7 @@ const HelloWorld: {[isoLangCode: string]: string} = {
   be: 'Прывітанне свет',
   bn: 'স্বাগতম বিশ্ব',
   bs: 'Zdravo Svijete',
+  bw: 'Dumela Lefatshe'
```

Comment:
Wrong ISO code for setswana and missing comma.

Reply:
My bad, I just fixed It.